dockerfiles/base/scripts/base/commands/cmd_config.sh

```diff
@@ -76,7 +82,21 @@ generate_configuration_with_puppet() {
     CHE_ENV_FILE="${CHE_HOST_INSTANCE}/config/$CHE_MINI_PRODUCT_NAME.env"
   fi
 
-  if [ "${CHE_DEVELOPMENT_MODE}" = "on" ]; then
+  if debug_server; then
+    DEV_MODE="development"
+    WRITE_LOGS=""
```

Comment:
Yes, it means that puppet logs will be streamed if you have `--debug` only.  If you mount a repository, but do not provide `--debug` then they would not be streamed.